Builds failing with template imports not found

After updating to 2.0.1 and 2.0.2, build keeps failing with 
```
Module not found: Error: Can't resolve './template' in '$TMPDIR\embroider\864ay1\Project\node_modules\ember-table\components\-private\scroll-indicators\component.js'
```

in `scroll-indicators\component.js` the `ember-table` is doing:
```
import layout from './template';
```
embroider creates `$TMPDIR\embroider\864ay1\Project\node_modules\ember-table\components\-private\scroll-indicators\template.hbs.js` file with content like this:

```
import { precompileTemplate } from "@ember/template-compilation";
export default precompileTemplate(" ...< actual hbs content of template.hbs> ...", {
  moduleName: "ember-table/components/-private/scroll-indicators/template.hbs"
});
```

but the file is not resolved at import. I imagine there is a problem with changes related to the babel precompilation plugin and the changed .hbs extensions somewhere?


embroider v2.0.2 on Windows, every package at latest as of now.

EDIT: after changing the import line to 
```
import layout from './template.hbs';
```
the build passed.

--- a/packages/compat/src/compat-app.ts
+++ b/packages/compat/src/compat-app.ts
@@ -232,7 +232,7 @@ class CompatAppAdapter implements AppAdapter<TreeNames> {
     // For TS, we defer to ember-cli-babel, and the setting for
     // "enableTypescriptTransform" can be set with and without
     // ember-cli-typescript
-    return ['.wasm', '.mjs', '.js', '.json', '.hbs', '.ts'];
+    return ['.wasm', '.mjs', '.js', '.json', '.hbs', '.hbs.js', '.ts'];
   }
 
   private *emberEntrypoints(htmlTreePath: string): IterableIterator<Asset> {

